example.py

```diff
@@ -0,0 +1,15 @@
+from nameko.rpc import rpc
+from nameko_twilio import Twilio
+
+
+class SmsService:
+    name = "sms_service"
+    twilio = Twilio()
+
+    @rpc
+    def send_message(self, message, to):
+        self.twilio.messages.create(
+            to=to,
+            from_="+441293344603",
```

Comment:
This needs to be removed as it is sensitive data unique to our Twilio account.
When you make a twilio account it gives you a phone number which texts are sent from. This is that number above.
You should make a new config variable for this and have it assigned when you create the client in the dependency provider like we do with account_sid and auth_token.
We do have an issue here though which is that you then would no longer have access to this in the service class...
We'll have to have another call to discuss how to fix this but for the initial version, please just change this line to `from_="YOUR TWILIO NUMBER HERE"`